packages/netskope/manifest.yml

```
@@ -0,0 +1,74 @@
+format_version: 1.0.0
+name: netskope
+title: "Netskope"
+version: 0.1.0
+license: basic
+description: Collect logs from Netskope with Elastic Agent.
+type: integration
+categories:
+  - security
+release: beta
+conditions:
+  kibana.version: ^7.16.2 || ^8.0.0
```

```suggestion
  kibana.version: ^7.17.0 || ^8.0.0
```

Let's go with the final release of 7.x.